server/fleet/orbit.go

```diff
@@ -6,9 +6,11 @@ import "encoding/json"
 // fleetd (orbit) so that it can run commands or more generally react to this
 // information.
 type OrbitConfigNotifications struct {
-	RenewEnrollmentProfile  bool `json:"renew_enrollment_profile,omitempty"`
-	RotateDiskEncryptionKey bool `json:"rotate_disk_encryption_key,omitempty"`
-	NeedsMDMMigration       bool `json:"needs_mdm_migration,omitempty"`
+	RenewEnrollmentProfile      bool   `json:"renew_enrollment_profile,omitempty"`
+	RotateDiskEncryptionKey     bool   `json:"rotate_disk_encryption_key,omitempty"`
+	NeedsMDMMigration           bool   `json:"needs_mdm_migration,omitempty"`
+	NeedsWindowsMDMEnrollment   bool   `json:"needs_windows_mdm_enrollment,omitempty"`
```

Comment:
Should it be called `NeedsProgrammaticWindowsMDMEnrollment`? I.e. do we expect to have other similar notifications for different types of Windows enrollment?

Reply:
I would vote to use `NeedsProgrammaticWindowsMDMEnrollment`, yes. There are a bunch of different ways to enroll in windows (there's a whole spreadhseet about that 😂 ) so it's better to err on the side of being explicit.